src/main/java/seedu/modtracker/Parser.java

```diff
@@ -44,11 +41,29 @@ public void parse(String input, ModuleList modList, String name, Storage storage
         case COMMAND_DELETEEXP:
             modList.deleteExp(input, toPrint, storage);
             break;
+        case COMMAND_ADDTIME:
+            try {
+                modList.addTime(input, toPrint, storage);
+            } catch (Exception e) {
+                ui.printErrorMessage(e.getMessage());
+                System.out.println("");
+            }
+            break;
         case COMMAND_MINUS:
-            modList.minusTime(input, toPrint, storage);
+            try {
+                modList.minusTime(input, toPrint, storage);
+            } catch (Exception e) {
+                ui.printErrorMessage(e.getMessage());
+                System.out.println("");
+            }
             break;
         case COMMAND_LIST:
-            ui.printTable(modList, Integer.parseInt(command[1]));
+            try {
```

Comment:
I think it is better to implement exception handling inside the respective methods. But I believe this method works for now to prevent the app from crashing due to the exceptions.